raster-foundry/raster-foundry
# Don't use PREDICTOR=2 creation option for 64 bit imagery

## Overview

This PR removes a creation option from the COG command when the reported data type is `float64`. The only other 64-bit data type rasterio includes is complex64, and for simplicity I'm assuming that we don't have any idea what to do with complex data, so I'm not going to pretend.

### Checklist


### Notes

Didn't know I could splat inside a list literal 🎉 

## Testing Instructions

- build batch container
- bring up servers and Groundwork
- upload the tif from the linked issue into groundwork
- get your upload id
- `./scripts/console batch 'rf process-upload <your upload id>'`
- it should succeed and you should have tiles in GW

Closes azavea/raster-foundry-platfrom#1128
